### DISCUSSION TITLE
در حال حاضر کدام پروتکل ها دیرتر از بقیه فیلتر میشن توی ایران؟

### DISCUSSION THREAD

**Discussion by mycurelights** [hiddify/Hiddify-Manager#371] [Q&A سوالات رایج]
متاسفانه استفاده از cloudflare برای ما به خوبی جواب نمیده و سرعتها خیلی کند هست و پیدا کردن آی پی که برای همه 3-4 نفر کار کنه خیلی مشکله  برای همین از Direct استفاده میکنم و همه پروتکل ها رو غیرفعال کردم به جز Trojan و Vless اما از یکی دو روز تا چند ساعت بیشتر طول نمی کشه که آی پی رو بلاک میکنن. از این سرور فقط 3-4 نفر بیشتر استفاده نمیکنن. می خواستم ببینم که مطمئن ترین پروتکل ها کدوم هستن که احتمال فیلترشدنشون بیاد پایین.

**Comment by zZedix** [comment]
1 - trojan
2 - vless
ولی شما میتونی از cdn استفاده کنی ! دنبال ip هم نمیخواد بگردی ! بجای ip بزن discord.com

**Reply by mycurelights** [reply]
ممنون از راهنمایی. اتقاقا قبلا این کار رو کردم اما جواب نداد. شاید من جایی اشتباه کردم. دوباره امتحان میکنم. البته اگه ممکنه بگید فلسفه استفاده از discord.com به جای نزدیکترین آی پی چیه؟ 
 و اینکه برای trojan و vless همه اشون رو فعال کنم اویکه یعنی ws , tcp, grpc, xtls یا اینکه بعضی هاشون باعث لو ررفتن میشه ؟
